modules/ROOT/pages/install-helm.adoc

```diff
@@ -0,0 +1,177 @@
+= Installing Runtime Fabric on Self-Managed Kubernetes Using Helm
+
+[IMPORTANT]
+====
+Installing Runtime Fabric on Self-Managed Kubernetes requires tasks that must be performed by both IT administrators and MuleSoft organization administrators.
+====
+
+You can use Helm to install Anypoint Runtime Fabric on an Amazon Elastic Kubernetes Service (Amazon EKS), Azure Kubernetes Service (AKS), or Google Kubernetes Engine (GKE) installation that you manage. 
+
+== Steps to Install Runtime Fabric Using Helm
+
+When you use Helm to install Runtime Fabric, you:
+
+. Create a Runtime Fabric using Runtime Manager
+. Create a namespace for Runtime Fabric and
+. Create a Docker pull secret for pulling the Runtime Fabric component images
+. Add the Runtime Fabric Helm repository
+. Download the `values.yml` file and install Runtime Fabric
+. Complete install steps
+
+
+== Before you Begin
+
+Before installing Anypoint Runtime Fabric in a self-managed Kubernetes environment, ensure that you have:
+
+* Reviewed the architecture and requirements outlined in xref:index-self-managed.adoc[Runtime Fabric on Self-Managed Kubernetes].
+* xref:install-self-managed-network-configuration.adoc[Configured your network to support Runtime Fabric on Self-Managed Kubernetes].
+* Installed and configured your Kubernetes environment as follows:
++
+** Running an EKS, AKS, or GKE Kubernetes environment. Other Kubernetes environments are not supported.
+** Running a xref:index-self-managed.adoc#kubernetes-support[supported Kubernetes version].
+** Running an ingress controller to send external requests to applications.
+* Installed Helm 3 or later and have privileged user permissions.
+
+[NOTE]
+--
+If you use Helm to manage your Runtime Fabric installation, you can use the `rtfctl` command line utility later to manage Mule license keys and proxies. However, you’ll need to track any post-installation changes you make, as explained in the xref:upgrade-helm.adoc[upgrade documentation]. 
+--
+
+=== Helm Support 
+
+Runtime Fabric supports installations, upgrades, and rollbacks using Helm for versions 1.13 and later. 
+
+== Create a Runtime Fabric Using Runtime Manager
+
+To install Runtime Fabric on Self-Managed Kubernetes with Helm, first create a Runtime Fabric using Runtime Manager. This is required to obtain the activation data that is needed during installation.
+
+. From Anypoint Platform, select Runtime Manager.
+. Click *Runtime Fabrics*.
+. Click *Create Runtime Fabric*.
+. Enter the name of the new Runtime Fabric, then select one of the following options:
+
+* Amazon Elastic Kubernetes Service
+* Azure Kubernetes Service
+* Google Kubernetes Engine
+. Review the Support responsibility disclaimer, and if you agree, click *Accept*.
+. Click Helm.
+. Save the activation data captured in the code example provided by Runtime Manager. by Runtime Manager. You will need this activation data to install Runtime Fabric.
```

Comment:
We can see to follow steps as on Helm tab on UI we wont directly see activationData.